__tests__/pet.test.js

```diff
@@ -0,0 +1,281 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
+    it('returns an object', () => {
+      expect(new Pet('Fido')).toBeInstanceOf(Object);
+    });
+
+    it('sets the name property', () => {
+        const pet = new Pet('Fido');
+        expect(pet.name).toEqual('Fido');
+      });
+
+    it('has an initial age of 0', () => {
+        const pet = new Pet('Fido');
+        expect(pet.age).toEqual(0);
+    });
+
+    it('has an initial hunger of 0', () => {
+        const pet = new Pet('Fido');
+        expect(pet.hunger).toEqual(0);
+    });
+  });
+
+describe('growUp', () => {
+
+    it('increase the age by 1', () => {   
+        const pet = new Pet('Fido');
+        pet.growUp();
+        expect(pet.age).toEqual(1);
+    });
+
+    it('increase the hunger by 5', () => {
+        const pet = new Pet('Fido');
+        pet.growUp();
+        expect(pet.hunger).toEqual(5);
+    });
+
+    it('decrease the fitness by 3', () => {
```

Comment:
Could a test be added here to make sure the value for fitness does not go outside of the allowed limits? You could do this by setting the fitness level to just inside the threshold then calling the `growUp` function and verifying that it hasnt gone outside its allowed range